### ISSUE TITLE
-p parameter doesnt really work as expected

### ISSUE THREAD

**Issue by LMS007** [foreverjs/forever#782]
According to the readme.

-p  PATH         Base path for all forever related files (pid files, etc.)

The only stuff that gets moved into the provided path is the .log files, but the pids, sock and config.json files still get dumped into ~/.forever.  This is a problem for me because I have several servers which share these home dirs (due to remote account) and the pids are overwriting each other.

I also don't know what version I'm on. See https://github.com/foreverjs/forever/issues/781

**Comment by LMS007** [comment]
I forced a new version directly from github and things seem to be working now, 

"forever": "https://github.com/foreverjs/forever/archive/0.15.1.tar.gz",

the root issue seems to be #781

**Comment by LMS007** [comment]
I was mistaking about this working in the newer version. The config .json file worked to launch the script, but the parameter "-p" still does not work either passed in via command line or put into the json file.

**Comment by xFairs** [comment]
re

**Comment by cms5x** [comment]
I agree with LMS007 that I expected the `-p` switch to accomplish what I had to use FOREVER_ROOT=/path/to/.forever forever list (and other forever actions).
